shared/src/lib.rs

```diff
@@ -132,6 +132,23 @@ impl ConnId {
     }
 
     /// The CBOR encoding of the identifier.
+    ///
+    /// For short the 48 compact connection identifiers, this is identical to the slice
```

Comment:
> For short the 48 compact connection identifiers

I don't understand this part.

Reply:
Maybe just replace it by `For compact connection identifiers (integer value in -24..=23)`. Referring to the range is easier to track to the RFC if needed.

Reply:
Good point, fix forcepushed.